src/core/stream.h

```diff
@@ -142,7 +142,6 @@ typedef union QUIC_STREAM_FLAGS {
         BOOLEAN ReceiveMultiple         : 1;    // The app supports multiple parallel receive indications.
         BOOLEAN ReceiveFlushQueued      : 1;    // The receive flush operation is queued.
         BOOLEAN ReceiveDataPending      : 1;    // Data (or FIN) is queued and ready for delivery.
-        BOOLEAN ReceiveCallActive       : 1;    // There is an active receive to the app.
```

Comment:
As the comment at the top of this struct states, please update the copy of this in `quictypes.h`.